test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -0,0 +1,215 @@
+using Microsoft.ML.Core.Data;
```

Comment:
Header please

Reply:
Done

---
In reply to: [220398783](https://github.com/dotnet/machinelearning/pull/942#discussion_r220398783) [](ancestors = 220398783)